Connect.vb

```diff
@@ -0,0 +1,114 @@
+Imports System.Windows.Forms
+Imports System.Runtime.InteropServices
+Imports Extensibility
+Imports Microsoft.Office.Core
+Imports Microsoft.Office.Interop
+Imports Microsoft.Vbe.Interop
+
+
+''' <summary>
+''' Each VBA editor add-in made by vb.net class library project has same template only the change in GUID(Which identified the add-in uniquely) and program id 
+'''So this his is the template of an add-in for an VB editor. i.e what happened when Startup is complete or OnConnection or OnDisconnection or OnUpdate or OnBeginShutDown.
+'''When Connection is made we initialize the add-in by adding one button in the menu-bar and after disconnection we will delete the button from there otherwise what happen
+'''our add-in got disconnected and button remains there so when the connection made next time our add-in will make same button so at the end our menu bar get conjusted with the same name button.
+''' </summary>
+
+<ComVisible(True), Guid("BDA9ECFF-0EDE-4C1D-81D1-51F6B4FF5F50"), ProgId("MyVBAAddin.Connect")>

```

Comment:
Change the name of the add in.

Reply:
It is the program id not the add-in name. But it can be done in registry and i did that.